hs_core/management/commands/manage_task_nightly.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+
+"""
+Clears out two-day-old zip files from iRODS storage
+Meant to be run once a day via Jenkins
+"""
+
+from xml.etree import ElementTree
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.core.management.base import BaseCommand
+from rest_framework import status
+import requests
+
+from hs_core.hydroshare.resource import get_activated_doi, get_resource_doi, \
+    get_crossref_url, deposit_res_metadata_with_crossref
+from hs_core.models import BaseResource
+from theme.models import UserQuota, QuotaMessage, User
+from theme.utils import get_quota_message
+
+import logging
+
+
+class Command(BaseCommand):
+    help = "Clears out two-day-old zip files from iRODS storage."
```

Comment:
@aphelionz Again, this help text needs to be updated to apply to this management command.